src/main/java/com/github/redouane59/twitter/helpers/AbstractRequestHelper.java

```diff
@@ -29,19 +25,20 @@
 
   private final TwitterCredentials twitterCredentials;
   private final OAuth10aService service;
+  protected final TweetStreamConsumer tweetStreamConsumer;
   
-  public AbstractRequestHelper(TwitterCredentials twitterCredentials) {
+  protected AbstractRequestHelper(TwitterCredentials twitterCredentials) {
 	  this(twitterCredentials, new ServiceBuilder(twitterCredentials.getApiKey())
 			  .apiSecret(twitterCredentials.getApiSecretKey())
 			  .build(TwitterApi.instance()));
   }
   
-  public AbstractRequestHelper(TwitterCredentials twitterCredentials, OAuth10aService service) {
+  protected AbstractRequestHelper(TwitterCredentials twitterCredentials, OAuth10aService service) {
 	  this.twitterCredentials = twitterCredentials;
 	  this.service = service;
+    this.tweetStreamConsumer = new TweetStreamConsumer();
```

Comment:
Can you instantiate it directly outside the constructor ? 
see https://stackoverflow.com/a/3919225/2337334